api/openapi/v2/core-data.yaml

```diff
@@ -117,7 +115,7 @@ components:
     EventCountResponse:
       allOf:
         - $ref: '#/components/schemas/BaseResponse'
-      description: "Returns an aggregate event count. If the deviceId property is blank then the count is applicable to all events in the database. Otherwise, the count is applicable to the specified device."
+      description: "Returns an aggregate event count in uint32 integer type. If the deviceId property is blank then the count is applicable to all events in the database. Otherwise, the count is applicable to the specified device."
```

Comment:
You made some changes in API document last time.  Please include them, for example, you removed deviceId in the EventCountResponse

Reply:
As discussed in this [comment](https://github.com/edgexfoundry/edgex-go/pull/2476/files#r406276432), the path `/reading/count/device/{deviceId}` will be added in the OpenAPI swagger file.